src/GeositeFramework/js/Map.js

```diff
@@ -101,20 +100,24 @@ require(['use!Geosite',
         view.model.on('change:extent', function () {
             var currentExtent = view.model.get('extent');
 
-            if (!_.isEqual(currentExtent, view.esriMap.extent)) {
+            if (!_.isEqual(currentExtent, view.esriMapView.extent)) {
                 loadExtent(view);
             }
         });
 
+        /*
         // Configure the esri proxy, for (at least) 2 cases:
         // 1) For WMS "GetCapabilities" requests
         // 2) When it needs to make an HTTP GET with a URL longer than 2000 chars
         esri.config.defaults.io.proxyUrl = "proxy.ashx";
```

Comment:
This got effectively commented out, but it's hard to see because of mixed comment style.  If it was unintentional, you may need to restore the proxy setting, or if it was meant to be commented out, can you convert to just one style so that it's easy to catch and put in the searchable comment text about the support in 4.2 so we can track it?